spec/unit/shoulda/matchers/active_record/validate_uniqueness_of_matcher_spec.rb

```diff
@@ -450,9 +450,20 @@
 
         expect(record).to validate_uniqueness
       end
+
+      context 'when the value has no case sensitivity' do
+        it 'accepts', focus: true do
+          model = define_model_validating_uniqueness(
+            attribute_type: :string,
+            validation_options: { case_sensitive: true },
+          )
+          record = create_record_from(model, attribute_name => '123')
+          expect{validate_uniqueness.matches?(record)}.to raise_error Shoulda::Matchers::ActiveModel::CouldNotValidateCaseError
```

Comment:
Line is too long. [127/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.